python/python/lance/dataset.py

```diff
@@ -3854,6 +3854,41 @@ def list(self) -> dict[str, Tag]:
         """
         return self._ds.tags()
 
+    def get_version(self, tag: str) -> Optional[int]:
+        """
+        Get the version of a specific tag by name.
+
+        Parameters
+        ----------
+        tag: str
+            The name of the tag to retrieve.
+
+        Returns
+        -------
+        int or None
+            The version number of the tag if it exists, otherwise None.
+        """
+        return self._ds.get_version(tag)
+
+    def list_ordered(self, order: Optional[str] = None) -> list[str, Tag]:
+        """
+        List all dataset tags.
+
+        Parameters
+        ----------
+        order: str, optional
+            The order in which to return the tags.
+            "asc" or "desc" can be used to specify the order explicitly.
+
+        Returns
+        -------
+        list[str, Tag]
+            A list of tuples mapping tag names to version numbers.
+            order by 'version' if specified, default descending.
```

Comment:
Looks like this part of the doc was not updated accordingly, it's not "order by 'version' if specified", but it's always ordered? What about the following:

```
"""
        List all dataset tags ordered by the tag's corresponding version.

        Parameters
        ----------
        order: str, optional
            The order in which to return the tags, `asc` or `desc`, default to `desc`.

        Returns
        -------
        list[str, Tag]
            An ordered list of tuples mapping tag names to its `Tag` metadata. 
"""